src/libraries/Common/tests/TestUtilities/System/AssertExtensions.cs

```diff
@@ -45,6 +45,12 @@ public static void Throws<T>(Action action, string expectedMessage)
             Assert.Equal(expectedMessage, Assert.Throws<T>(action).Message);
         }
 
+        public static void ThrowsContains<T>(string paramName, Action action, string expectedMessageContent)
+            where T : ArgumentException
+        {
+            Assert.Contains(expectedMessageContent, Assert.Throws<T>(paramName, action).Message);
```

Comment:
1. I suppose? Do we really need a helper to turn a two-liner in to a one-liner that I am going to have to look up every time?
2. Should this defer to our own `AssertExtensions.Throws` instead of Xunit's? Seems like we prefer our own.

Reply:
> I vaguely feel like we had the parameter-name-verifying one first

Perhaps? We also have a variation of it to split the parameter name between .NET Framework and .NET Core.